### ISSUE TITLE
Request/Question: Can I pause a render while running, and then resume it later?

### ISSUE THREAD

**Issue by isaiahshiner** [overviewer/Minecraft-Overviewer#1415]
TLDR: If I Ctrl + C to stop the process, and then rerun it later with no changes to the config or output file, will the render resume where it left off? If this is not possible, could a pause key/command be added? Could I add this feature with a pull request?

I'm doing a render on a very large world, about 4GB. I did not want to crop the world because I want to see the full extent of the world. I'm about 4 hours in and there are about 7 hours left. I started it in the middle of the night while no one was on, but now people want to join and the server can't handle the render and MC (only 2 cores at 2.4 GHz). Oh and Ubuntu Linux VPS, if it matters.

**Comment by CounterPillow** [comment]
>TLDR: If I Ctrl + C to stop the process, and then rerun it later with no changes to the config or output file, will the render resume where it left off?

yes.

Overviewer will show a wrong ETA when resuming an aborted render, but it'll happily continue where it left off in `--check-tiles` mode.
